public/plugin.tsx

```diff
@@ -102,12 +103,14 @@ interface PublicConfig {
 export const [
   getRenderAccelerationDetailsFlyout,
   setRenderAccelerationDetailsFlyout,
-] = createGetterSetter('renderAccelerationDetailsFlyout');
+] = createGetterSetter<(acceleration: any) => void>('renderAccelerationDetailsFlyout');
```

Comment:
nit. add type for acceleration?

Reply:
@RyanL1997 Can you please edit this when we have a concrete structure for the input?